e2e/test/scenarios/sharing/public-dashboard.cy.spec.js

```diff
@@ -230,6 +230,21 @@ describe("scenarios > public > dashboard", () => {
     });
 
     // new dashboards should default to 'fixed' width
+    cy.findByTestId("fixed-width-dashboard-title").should(
+      "have.css",
+      "max-width",
+      "1048px",
+    );
+    cy.findByTestId("fixed-width-dashboard-tabs").should(
+      "have.css",
+      "max-width",
+      "1048px",
+    );
+    cy.findByTestId("fixed-width-filters").should(
+      "have.css",
+      "max-width",
+      "1048px",
+    );
```

Comment:
We seem to be duplicating these assertions, and the width value itself pretty often
Let's extract an E2E helper that would do that, so we have a single place to update when we need it

Could be something like `assertDashboardWidth` / `assertFixedDashboardWidth`
We can have a similar helper for `not.have.css` assertions below